### PR TITLE
sci-* EAPI-5 cleanup

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: sci-chemistry/bodr/bodr-10.ebuild [hunk 1]
```diff
@@ -10,14 +10,11 @@ SRC_URI="mirror://sourceforge/${PN}/${P}.tar.bz2"
...
-DEPEND="dev-libs/libxslt"
-RDEPEND=""
-
-DOCS=( "THANKS" "AUTHORS" "README" "ChangeLog" )
+BDEPEND="dev-libs/libxslt"
 
 src_prepare() {
+	default
 	sed -i -e "s/COPYING//g" Makefile.* || die
 }
```
Comment (thesamesam) [inline sci-chemistry/bodr/bodr-10.ebuild:20]: We can drop the [docdir](https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-128001r8) in newer EAPIs (yay!)

File: sci-libs/qrupdate/qrupdate-1.1.2-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+src_compile() {
+	emake solib
+}
+
+src_install() {
+	emake DESTDIR="${D}" install-shlib
+	dosym libqrupdate.so.$(ver_cut 1) /usr/$(get_libdir)/libqrupdate.so
+	dodoc README ChangeLog
```
Comment (SoapGentoo) [inline sci-libs/qrupdate/qrupdate-1.1.2-r2.ebuild:49]: check whether `einstalldocs` maybe also installs additional stuff?

**Review by SoapGentoo — APPROVED** [review]

File: sci-libs/qrupdate/qrupdate-1.1.2-r2.ebuild (hunk 2 above)
Comment (jsmolic) [reply sci-libs/qrupdate/qrupdate-1.1.2-r2.ebuild:49]: nope, only readme and changelog

File: sci-chemistry/bodr/bodr-10.ebuild (hunk 1 above)
Comment (jsmolic) [reply sci-chemistry/bodr/bodr-10.ebuild]: Perfect :smiley:

**Pull request closed without merging**